code/game/objects/items/natural/animals.dm

```diff
@@ -83,6 +83,7 @@
 	grid_width = 64
 	headpricemin = 3
 	headpricemax = 7
+	sellprice = 5
```

Comment:
price is too low . this is not a guaranteed item . at least 30 so the merchant buys it for 10 - 15